actions_helper/main.py

```diff
@@ -43,11 +43,15 @@ def cmd_dependency_update(dry_run: str):
 
 #### Not Updated
 
-After merging this pull request, the following packages will **not** be on the latest version:
+{
+f"""After merging this pull request, the following packages will ** not ** be on the latest version:
```

Comment:
```suggestion
f"""After merging this pull request, the following packages will **not** be on the latest version:
```

Is there any specific reason for adding the empty spaces here?

Reply:
No specific reason